capa/features/extractors/viv/helpers.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 FireEye, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+from vivisect.const import REF_CODE
+
+
+def get_coderef_from(vw, va):
+    """
+    return first code `tova` whose origin is the specified va
+    vivisect xref tuple: (fromva, tova, reftype, rflags)
+    """
+    xrefs = vw.getXrefsFrom(va, REF_CODE)
+    if len(xrefs) > 0:
+        return xrefs[0][1]
```

Comment:
the constants are found here: https://github.com/vivisect/vivisect/blob/master/vivisect/const.py#L164 and would avoid the need to provide a comment explaining what the magic `1` is.

Reply:
The `1` is for the tuple entry `tova` documented in the doc string. How can I clarify this?

Reply:
ah, gotcha now :) thanks, I'll do that